README.md

```diff
@@ -47,8 +47,21 @@ Install pip for python3
 1. Login the admin panel and create entries in the Navbar Entries section as per your requirement.
 2. Load and use the `navbar` tag if required in any of your custom pages.
 
+### Setting up email templates
+1. Create templates in both text and html formats.
+2. Place them in the directory cited below.
```

Comment:
As of now, it just works that way, we would need to manually create files in templates.
I too think it's a lame way.

I think it's better to put up some use to WYSIWYG editor with a form that automates this stuff.
But again, how do we put the `context` stuff into the mail?